app/uk/gov/hmrc/individualsemploymentsapi/controller/v1/CommonControllerV1.scala

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.individualsemploymentsapi.controller.v1
+
+import org.joda.time.DateTime
+import play.api.Logger
+import play.api.mvc.{ControllerComponents, Request, RequestHeader, Result}
+import uk.gov.hmrc.auth.core.authorise.Predicate
+import uk.gov.hmrc.auth.core.retrieve.v2.Retrievals
+import uk.gov.hmrc.auth.core.{AuthorisationException, AuthorisedFunctions, Enrolment}
+import uk.gov.hmrc.http.{HeaderCarrier, TooManyRequestException}
+import uk.gov.hmrc.individualsemploymentsapi.audit.v2.AuditHelper
+import uk.gov.hmrc.individualsemploymentsapi.controller.v1.Environment.SANDBOX
+import uk.gov.hmrc.individualsemploymentsapi.error.ErrorResponses._
+import uk.gov.hmrc.individualsemploymentsapi.util.Dates._
+import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
+
+import javax.inject.Inject
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+abstract class CommonControllerV1 @Inject()(cc: ControllerComponents) extends BackendController(cc) {
```

Comment:
Can we please remove the version from the file name and class name as its already described in the package?